packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -167,6 +182,37 @@ const useTransactionHistoryWithoutStatuses = (address: string | undefined) => {
   const [depositsLoading, setDepositsLoading] = useState(true)
   const [withdrawalsLoading, setWithdrawalsLoading] = useState(true)
 
+  const cctpTransfersMainnet = useCctpFetching({
```

Comment:
This isn't scalable with potentially more CCTP supported chains in the future, but for now it's the simplest solution.

Long term we'd need to make changes to CCTP hooks which would be quite a bit.